doc/api/child_process.md

```diff
@@ -1398,6 +1398,23 @@ then this will be `null`.
 `subprocess.stdout` is an alias for `subprocess.stdio[1]`. Both properties will
 refer to the same value.
 
+```js
+const { spawn } = require('child_process');
+
+const subprocess = spawn('ls');
+
+const ioStdout = subprocess.stdio[1];
+
+// This is the same of listening to ioStdout data event
```

Comment:
@Trott I've made this change by receiving your review! I think this is quite better, what do you think?

Reply:
The key point isn't that listening on the `data` event on `subprocess.stdout` is the same as lstening on the `data` event on `ioStdout`. The key point is that `subprocess.stdout` is `ioStdout`. They are exactly the same thing.

I would just remove all the `ioStdout` stuff.

Reply:
You're right, It's more clear without `ioStdout` stuff, fixed on [cfcf142](https://github.com/nodejs/node/pull/28402/commits/cfcf142c7d2c75cc3776978309f59d403fa675c1)